### DISCUSSION TITLE
--Train_Text_Encoder variation

### DISCUSSION THREAD

**Discussion by AbyszOne** [smy20011/dreambooth-gui#56] [Q&A] [answered]
I have noticed that --train_text_encoder has a significant effect on training. However, it is very aggressive, and I have read that on some pages its percentage can be varied. Is that possible? Is there such a command? It would be really important.

**Comment by futurevessel** [comment, marked as answer]
Looking at TheLastBen colab (https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast-DreamBooth.ipynb), the parameter used to control this is '--stop_text_encoder_training='

However the value the parameter takes is not a percentage, so the colab calculates this as follows: https://github.com/TheLastBen/fast-stable-diffusion/commit/3e28d1a13508524872e4aca8fb30e657bb055ed6

Seems like a good option to have, as just as described in the commit, it's easy to overfit when using --train_text_encoder

**Reply by AbyszOne** [reply]
Thanks!. A doubt. There is a comand there "Train_text_encoder_for=" wich is for %. Can i use that in the parameters or i should use "--stop..." to put the number of steps manually?

**Reply by futurevessel** [reply]
No, "Train_text_encoder_for=" is just part of the colab gui, the actual parameter is '--stop_text_encoder_training=', but I don't know if dreambooth-gui supports it. If I'm not mistaken dreambooth-gui is based upon ShivamShrirao's repo, and I don't know if that supports --stop_text_encoder_training

I'm currently running a training so I can't try it myself.
